.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,23 @@
+{
+	"name": "Go",
+	"dockerFile": "Dockerfile",
+	"appPort": 9000,
+	"extensions": [
+		"ms-vscode.go"
+	],
+	"runArgs": [
+		// Comment out the next line to run as root instead. Linux users, 
+		// update Dockerfile with your user's UID/GID if not 1000.
+		"-u",
+		"circleci",
```

Comment:
The comment should now say to comment out the next 2 lines instead of just the one to avoid confusion.